modules/doc/content/getting_started/installation/moose_conda_binary.md

```diff
@@ -0,0 +1,153 @@
+# Pre-Built MOOSE
+
+!style! halign=left
+If you are not interested in developing your own MOOSE based application, and wish to use MOOSE's
+many available physics solvers, you can install the pre-built fully-featured MOOSE binary.
+!style-end!
+
+## Install Conda
+
+!style! halign=left
+Our preferred method for deliverying pre-built MOOSE binaries is via Conda.
+!style-end!
+
+!include getting_started/installation/install_miniconda.md
+
+## Install MOOSE
+
+!style! halign=left
+With Conda initialized, create the `moose` environment and install `moose`:
+!style-end!
+
+```bash
+mamba create -n moose moose
+```
+
+After the installation completes, activate the new environment:
+
+```bash
+mamba activate moose
+```
+
+!alert note
+Know that you will need to `mamba activate moose` for +each terminal window you open, and each time
+you wish to perform MOOSE related work+. If you wish to make this automatic, you can add that
+command to the end of your shell profile.
+
+Verify `moose` is available:
+
+```bash
+moose --help
+```
+
+Should produce the help page. This simple command demonstrates that you have sucessfully installed
+the MOOSE Conda package.
+
+## Run an Example
+
+!style! halign=left
+A MOOSE installation binary comes with several examples you can run to make sure everything
+is sound, as well as moving some of the example inputs into a safe location you can play with.
+!style-end!
+
+There are examples for each physics solver available by name, in the following directory:
+
+```bash
+ls $CONDA_PREFIX/moose/share/combined
+```
+
+!alert! note
+Not everything you find in this directory is a physics library. We are working on an elegant way to
+ask `moose` for all available solvers.
+!alert-end!
+
+For now, lets copy the reactor module into a safe location for editing:
+
+```bash
+mkdir -p ~/projects/examples
+cd ~/projects/examples
+moose --copy-inputs reactor_workshop
+```
+
+!alert! note
+Take note of the information being displayed in the output. `moose` is alerting to the directory
+structure it created. Which can sometimes not represent the exact wordage you provided as arguments.
+!alert-end!
+
+With the reactor module's examples and inputs copied, move into the reactor workshop directory and
+instruct `moose` to run the tests:
+
+```bash
+cd combined/reactor_workshop
```

Comment:
it s odd to be using combined/reactor_workshop

why the 'combined' folder?

Reply:
Copied inputs go into `<binary name>/<installed name>`. This is a product of this process.